specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json

```diff
@@ -0,0 +1,1695 @@
+          "description": "Subscription ID.",
+          "type": "string"
+        },
+        "resourceType": {
+          "description": "Resource type.",
+          "type": "string"
+        },
+        "resourceLocation": {
+          "description": "Resource location.",
+          "type": "string"
+        },
+        "resourceGroup": {
+          "description": "Resource group name.",
+          "type": "string"
+        },
+        "resourceTags": {
+          "description": "List of resource tags.",
+          "type": "string"
+        },
+        "policyAssignmentName": {
+          "description": "Policy assignment name.",
+          "type": "string"
+        },
+        "policyAssignmentOwner": {
+          "description": "Policy assignment owner.",
+          "type": "string"
+        },
+        "policyAssignmentParameters": {
+          "description": "Policy assignment parameters.",
+          "type": "string"
+        },
+        "policyAssignmentScope": {
+          "description": "Policy assignment scope.",
+          "type": "string"
+        },
+        "policyDefinitionName": {
+          "description": "Policy definition name.",
+          "type": "string"
+        },
+        "policyDefinitionAction": {
+          "description": "Policy definition action, i.e. effect.",
+          "type": "string"
+        },
+        "policyDefinitionCategory": {
+          "description": "Policy definition category.",
+          "type": "string"
+        },
+        "policySetDefinitionId": {
+          "description": "Policy set definition ID, if the policy assignment is for a policy set.",
+          "type": "string"
+        },
+        "policySetDefinitionName": {
+          "description": "Policy set definition name, if the policy assignment is for a policy set.",
+          "type": "string"
+        },
+        "policySetDefinitionOwner": {
+          "description": "Policy set definition owner, if the policy assignment is for a policy set.",
+          "type": "string"
+        },
+        "policySetDefinitionCategory": {
+          "description": "Policy set definition category, if the policy assignment is for a policy set.",
+          "type": "string"
+        },
+        "policySetDefinitionParameters": {
+          "description": "Policy set definition parameters, if the policy assignment is for a policy set.",
+          "type": "string"
+        },
+        "managementGroupIds": {
+          "description": "Comma separated list of management group IDs, which represent the hierarchy of the management groups the resource is under.",
+          "type": "string"
+        },
+        "policyDefinitionReferenceId": {
+          "description": "Reference ID for the policy definition inside the policy set, if the policy assignment is for a policy set.",
+          "type": "string"
+        },
+        "complianceState": {
+          "description": "Compliance state of the resource.",
+          "type": "string"
+        },
```

Comment:
Looking at example, this also looks like an enum. Might be worth adding the valid values to make it easier for the user. 

If this compliance state and other one you're adding below is of the same type (same allowed values), then you should probably declare an enum type and use it in all the places where it fits.